husky_components/matrix/main.js

```diff
@@ -11,7 +11,8 @@ define(function() {
 
     'use strict';
 
-    var activeClass = 'is-active';
+    var activeClass = 'is-active',
+        selectButton = 'td:last-child > span:last-child';
```

Comment:
I have already written before that this variable is not representing a button, so rename it to what it really is: `selectButtonSelector`